utils/paste.tsx

```diff
@@ -66,8 +66,14 @@ export function formatMarkdownTableMessage(table: HTMLTableElement, message?: st
     }).join('\n');
 
     const formattedTable = `${header}${body}\n`;
-
-    return message ? `${message}\n\n${formattedTable}` : formattedTable;
+    if (!message) {
+        return formattedTable;
+    }
+    if (typeof caretPosition === 'undefined') {
+        return message ? `${message}\n\n${formattedTable}` : formattedTable;
```

Comment:
This check for `message` seems redundant given the new introduced `if` above ? 
```suggestion
        return ${message}\n\n${formattedTable}`;
```

Reply:
you're right! changing it right away